runtime/sydney/src/pos/mod.rs

```diff
@@ -90,12 +90,12 @@ parameter_types! {
 	pub const MaxCandidateIntake: u32 = 1;
 	pub const SocietyPalletId: PalletId = PalletId(*b"py/socie");
 
-	// Six sessions in an era (90 * 6 * 4 hours = 90 days eras).
-	pub const SessionsPerEra: SessionIndex = prod_or_fast!(90 * 6, 6);
+	// Six sessions in an era (6 * 4 hours = 1 day).
+	pub const SessionsPerEra: SessionIndex = prod_or_fast!(6, 6);
```

Comment:
> there is fast-runtime feature with the same values
> 
> Look at the definition of prod_or_fast!

It is because 90 days of an era is a long time for them to refresh their auto compound rewards.

Reply:
We need feedback of Matthew then. Cause it was requirement early to have a long eras.
It was at time of MPC adoption but still

Reply:
> It was at time of MPC adoption but still

MPC is going in a different direction, so this is no longer a concern.

Reply:
I think we can remove prod_or_fast macro for this fields as they are equal now.